File: leetcode/arrays/FactorialZeroes.java
public class FactorialZeroes {
    public static int factorialZeroes(int n){
        int count = 0;

        while(n > 0){
            n /= 5;
            count += n;
        }

        return count;
    }

    public static void main(String[] args) {
        int numTrailingZeroes = factorialZeroes(12);
        System.out.println(numTrailingZeroes);
    }
}

File: leetcode/arrays/factorialZeroes.py
class Solution:
	# @param A : integer
	# @return an integer
    def factorialZeroes(self, A):
        count = 0
        while (A > 0):
            A = int(A/5)
            count += A
            return count


if __name__ == "__main__":
    sol = Solution()
    ind = sol.factorialZeroes(12)
    print(ind)